allauth/urls.py

```diff
@@ -12,11 +12,14 @@
 if app_settings.SOCIALACCOUNT_ENABLED:
     urlpatterns += [url(r'^social/', include('allauth.socialaccount.urls'))]
 
+# Provider urlpatterns, as separate attribute (for reusability).
+provider_urlpatterns = []
 for provider in providers.registry.get_list():
     try:
         prov_mod = import_module(provider.get_package() + '.urls')
     except ImportError:
         continue
-    prov_urlpatterns = getattr(prov_mod, 'urlpatterns', None)
-    if prov_urlpatterns:
-        urlpatterns += prov_urlpatterns
+    prov_mod_urlpatterns = getattr(prov_mod, 'urlpatterns', None)
+    if prov_mod_urlpatterns:
+        provider_urlpatterns += prov_mod_urlpatterns
```

Comment:
Do not really have to rename `prov_urlpatterns` here - happy to revert those lines (I was exposing `prov_urlpatterns` initially, but then thought `provider_urlpatterns` is more explicit).